### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by Christgo**
Code Review Request
Make bEtTr PlEs
@StevenHowlett
@PatersonConnor

### CODE REVIEW

File: prac_03/password_entry.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def main():
     print_password(password_length)
```
Comment (StevenHowlett) [inline prac_03/password_entry.py]: this is from prac 3 why is it in prac 5

File: prac_05/word_occurences.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Counting word occurrences
+"""
+# TODO: Sort list so that highest word is at the top
+
+
+def main():
+    """Calculates and displays the word and its occurrences in a user string"""
+    user_words_to_occurrences = {}
+
+    user_word = input("Text:")
```
Comment (StevenHowlett) [inline prac_05/word_occurences.py]: user word implies that the users could only enter 1 word user text might be better

File: prac_05/word_occurences.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    user_words = user_word.split(" ")
```
Comment (StevenHowlett) [inline prac_05/word_occurences.py]: words are aren't used anywhere else maybe words would be a better variable

File: prac_05/word_occurences.py (hunk 2 above)
Comment (Christgo) [reply prac_05/word_occurences.py]: changed to user_string = input("Text:")

**Pull request merged**